tokens/tokens-eth.json

```diff
@@ -8092,6 +8092,31 @@
 			"youtube": "https://www.youtube.com/watch?v=UNGDGDDIz7Y"
 		}
 	},
+	{
+		"symbol": "LOCUS",
+		"address": "0xC64500DD7B0f1794807e67802F8Abbf5F8Ffb054",
```

Comment:
I found this token on etherscan with the same name: https://etherscan.io/address/0xcb3b65f59f96a60e4f8688ebf0587cbe93993a3e

Are you sure about the address in your PR?